File: c++/171_Excel_Sheet_Column_Number.cpp
#include <iostream>

using namespace std;

class Solution {
public:
	int titleToNumber(string s) {
		int ans = 0;
		for (int i = 0; i < s.size(); i++){
			ans = ans * 26 + s[i] - 'A' + 1;
		}
		return ans;
	}
};

int main(){
	Solution s;
	cout << s.titleToNumber("AB");
	return 0;
}

File: C#/171_Excel_Sheet_Column_Number.cs
using System;

namespace LeetCode
{
    public class Solution
    {
        public int TitleToNumber(string s)
        {
            int ans = 0;
            for (int i = 0; i < s.Length; i++)
            {
                ans = ans * 26 + s[i] - 'A' + 1;
            }
            return ans;
        }
        static void Main(string[] args)
        {
            Solution s = new Solution();
            Console.Write(s.TitleToNumber("AB"));
        }
    }
}
